### PR TITLE
Docs for new API endpoints

### PR DESCRIPTION
**Pull request by wonderseif**
To be merged after: https://github.com/wondercraft-ai/wondercraft-podcast-backend/pull/829

### Existing endpoint
image

### Voice IDs on existing endpoint 
image

### [NEW] User Scripted endpoint
image

image

### CODE REVIEW

File: api-reference/endpoint/delete.mdx [hunk 1]
```diff
@@ -1,4 +0,0 @@
----
-title: 'Delete Plant'
-openapi: 'DELETE /plants/{id}'
```
Comment (wonderseif) [inline api-reference/endpoint/delete.mdx:3]: Probably from the original template... 😄

File: api-reference/endpoint/user_scripted.mdx [hunk 2]
```diff
@@ -0,0 +1,4 @@
+---
+title: "Create an episode with a user provided script"
```
Comment (wonderseif) [inline api-reference/endpoint/user_scripted.mdx:2]: Open to other suggestions here

File: api-reference/openapi.json [hunk 3]
```diff
@@ -1,23 +1,27 @@
...
   "openapi": "3.1.0",
   "info": {
-    "title": "Wondercraft.ai API documentation",
-    "description": "You can find\nout more about Wondercraft.ai\n[here](https://www.wondercraft.ai/).\n",
-    "termsOfService": "https://support.wondercraft.ai/hc/en-gb/articles/14294433754258-Terms-of-Service",
-    "contact": { "email": "team@wondercraft.ai" },
-    "version": "0.1.0"
+    "title": "Wondercraft Public API",
+    "version": "0.0.1"
+  },
```
Comment (wonderseif) [inline api-reference/openapi.json:6]: lol - we can also keep 0.1.0.. 
I basically replaced this whole file with the `openapi.json` produced from the set of endpoints we only care about exposing.

File: api-reference/endpoint/user_scripted.mdx [hunk 4]
```diff
@@ -0,0 +1,4 @@
...
+openapi: "POST /api/v1/podcast/user-scripted"
```
Comment (youssefarizk) [inline api-reference/endpoint/user_scripted.mdx:3]: change this to `scripted`?

**Review by youssefarizk — APPROVED** [review]

**Pull request merged**